README.md

```diff
@@ -38,9 +39,9 @@ which is a human-readable form of the survey respondent's response to question.
     We recommend using a virtualenv:
 
     ```
-    virtualenv -p python3 venv
+    python3 -m venv venv
```

Comment:
Since `venv` repeating itself twice in this command, we can write it as below to avoid any confusion
```suggestion
    python3 -m venv ./venv
```

Reply:
If we're updating the instructions, we should try to be consistent with the rest of the Singer documentation.
> `python3 -m venv ~/.virtualenvs/<tap-foo>`
`source ~/.virtualenvs/<tap-foo>/bin/activate`

[Source](https://github.com/singer-io/getting-started/blob/b491ed311f93b0e4af94b369384cf83c3b5ceab8/docs/RUNNING_AND_DEVELOPING.md#running-a-singer-tap)